Index: CanvasStart.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- CanvasStart.py	(revision 0713f405303fee96d299cfe81bdd13cedc3546d3)
+++ CanvasStart.py	(date 1609597056946)
@@ -64,5 +64,4 @@
     root.mainloop()
 
 
-if __name__ == '__main__':
-    main()
\ No newline at end of file
+main()
\ No newline at end of file
Index: Voronoi.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Voronoi.py	(revision 0713f405303fee96d299cfe81bdd13cedc3546d3)
+++ Voronoi.py	(date 1609597230125)
@@ -91,8 +91,6 @@
         return not self.pq
 
 
-
-
 class Voronoi:
     def __init__(self, points):
         self.output = []
